unity/unity-sources/Runtime/Mono/unity-aot.props

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I was thinking a different file structure.  Something like
```
unity\unityaot-props\System.Private.CoreLib.props
unity\unityaot-props\System.Linq.props
```

It doesn't seem like the current organization would scale once we have to do this in another csproj?  Or am I missing how that would work?